Marlin/SdFatConfig.h

```diff
@@ -125,7 +125,11 @@
   * Defines for long (vfat) filenames
   */
   /** Number of VFAT entries used. Every entry has 13 UTF-16 characters */
-  #define MAX_VFAT_ENTRIES (2)
+  #if ENABLED(SCROLL_LONG_FILE_NAMES)
+    #define MAX_VFAT_ENTRIES (5)
```

Comment:
This is fine if FAT32 supports it. Just note that it balloons the memory requirements for `SD_ALPHA_SORT` and other functions that deal with long filenames.

Reply:
I found that without increasing that value, only enough characters to fit the display were kept, so nothing was gained by scrolling. In order to have something to scroll, I needed to increase MAX_VFAT_ENTRIES such that more characters were being maintained. I do not know if there would be a way to implement scrolling without that additional memory consumption.

Reply:
Perhaps if there was a way to request from the SD driver the complete filename for just the currently selected file, the memory footprint could be reduced as this could be done only when a file was selected in the LCD.

Reply:
The `SDCARD_ALPHA_SORT` feature has various options. So I'm just adding _one more_ to decide how much of the filename to keep when using RAM caching. When not using RAM caching, it works out to have a much smaller footprint, since it does just fetch the name on-demand. There's one additional feature which will use dynamic RAM allocation, in which case it only allocates exactly the number of bytes needed to hold the long filenames, and no more. This feature is contraindicated, except on processors like 1284p (and 32-bit) where there's plenty of extra SRAM to spare.